packages/es-client/esSearchAfter.js

```diff
@@ -0,0 +1,55 @@
+const { Search } = require('./search');
+
+/**
+ * ES search_after wrapper class for Search.
+ *
+ * This class just allows a user to access the search_after api of elasticsearch for
+ * long lived requests that require multiple server requests.
+ *
+ * Use: Instantiate a class with the queryStringParameters, type, and index.
+ * and call the object.query() until no more results are returned.
+ */
+class ESSearchAfter extends Search {
+  /**
+   * Build search params for search-after API.
+   *
+   * @returns {Object} ES search params
+   */
+  _buildSearch() {
+    const params = super._buildSearch();
+    delete params.from;
```

Comment:
We need to delete `params.from` (which is always passed in from the dashboard currently) because having `params.from` causes ES to do a normal search (which is capped at 10k) and overrides any `search_after` config. `params.to` is ignored when `from` is not provided, but we could delete it too to be explicit, I guess.